engines/pyncm/src/lib.rs

```diff
@@ -82,31 +61,15 @@ impl Engine for PyNCMEngine {
 async fn fetch_song_info(id: &str, ctx: &Context) -> anyhow::Result<PyNCMResponse> {
     debug!("Fetching the song information…");
 
-    let bitrate = if ctx.enable_flac { 999000 } else { 320000 };
+    let bitrate = if ctx.enable_flac { 999 } else { 320 };
     let url = Url::parse_with_params(
-        "https://pyncmd.apis.imouto.in/api/pyncm?module=track&method=GetTrackAudio",
-        &[("song_ids", id), ("bitrate", &bitrate.to_string())],
+        "https://music.gdstudio.xyz/api.php?types=url&source=netease",
```

Comment:
```suggestion
        "https://music-api.gdstudio.xyz/api.php?types=url&source=netease",
```

https://github.com/UnblockNeteaseMusic/server/issues/1395